Tests/PINRemoteImageTests.m

```diff
@@ -81,8 +109,10 @@ @interface PINRemoteImage_Tests : XCTestCase <PINURLSessionManagerDelegate>
 @property (nonatomic, strong) PINRemoteImageManager *imageManager;
 @property (nonatomic, strong) NSMutableData *data;
 @property (nonatomic, strong) NSURLSessionTask *task;
+@property (nonatomic, strong) PINRemoteImageDownloadTask *firstDownloadTask;
+@property (nonatomic, strong) PINRemoteImageDownloadTask *secondDownloadTask;
 @property (nonatomic, strong) NSError *error;
-
+@property (nonatomic, strong) dispatch_semaphore_t semaphore;
```

Comment:
Should the new properties be nil'd out in tearDown?

Reply:
🤔 We should.